### PR TITLE
Added number format accessor for legend chart

### PR DESCRIPTION
**Pull request by martinmanzo**
## Description
Related to https://github.com/eventbrite/britecharts/issues/295

## Motivation and Context
Fixes https://github.com/eventbrite/britecharts/issues/295

## How Has This Been Tested?
Included new test, added code to the demo file.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: docs/legend.js.html [hunk 1]
```diff
@@ -280,7 +280,7 @@ <h1 class="page-title">Source: legend.js</h1>
 
             getId = ({id}) => id,
             getName = ({name}) => name,
-            getFormattedQuantity = ({quantity}) => numberFormat(quantity),
+            getFormattedQuantity = ({quantity}) => d3Format.format(numberFormat)(quantity),
```
Comment (Golodhros) [inline docs/legend.js.html]: Awesome!

File: test/specs/legend.spec.js [hunk 2]
```diff
@@ -216,6 +216,18 @@ define(['d3', 'legend', 'donutChartDataBuilder'], function(d3, legend, dataBuild
                     expect(d3.select(elements[3]).attr('class')).toEqual('legend-entry');
                     expect(d3.select(elements[4]).attr('class')).toEqual('legend-entry');
                 });
+
+                it('should provide numberFormat getter and setter', () =>{
```
Comment (Golodhros) [inline test/specs/legend.spec.js]: Thanks for the test!

**Comment by martinmanzo** [comment]
Just updated with master and rebuilt the docs.

**Comment by cpuy** [comment]
super-happy-men

**Pull request merged**